cypress/e2e/common/common.js

```diff
@@ -996,6 +995,85 @@ Given("I seeded the database",() => {
   })
 })
 
+Given("I seeded the database with an asset {string} with no attached tenure", (assetGuid) => {
```

Comment:
Some tests required an asset with no tenure. This way you're guaranteed to see the 'New tenure' button and proceed with the creation of a new tenure.